pancancer_utilities/data_utilities.py

```diff
@@ -0,0 +1,99 @@
+import os
+
+import pandas as pd
+import pickle as pkl
+from sklearn.model_selection import KFold
+from sklearn.preprocessing import MinMaxScaler
+
+import pancancer_utilities.config as cfg
+
+def load_expression_data(subset_mad_genes=cfg.num_features_raw,
+                         scale_input=False,
+                         verbose=False):
+    """Load and preprocess saved TCGA gene expression data.
+
+    Arguments
+    ---------
+    subset_mad_genes (int): TODO still need to implement this
+    scale_input (bool): whether or not to scale the expression data
+    verbose (bool): whether or not to print verbose output
+
+    Returns
+    -------
+    rnaseq_df: samples x genes expression dataframe
+    """
+    if verbose:
+        print('Loading gene expression data...')
```

Comment:
I've thought about this in the past and generally stuck with stdout for log messages, since I was worried that other programs that might call my scripts would consider anything on stderr an error. But it looks like log messages on stderr is standard practice: https://unix.stackexchange.com/a/331808

I'm not sure if I'm planning to write results to stdout or directly to files, but I'll switch this to stderr for now like you suggested.